src/propagate_constant.cpp

```diff
@@ -36,6 +36,8 @@ bool skip_propogate(instruction_ref ins)
 {
     if(ins->name() == "contiguous")
         return skip_propogate(ins->inputs().front());
+    if(ins->name() == "convert" and ins->inputs().front()->can_eval())
```

Comment:
`ins->inputs().front()->can_eval()` should always be true.

However, we dont want to fold the convert if its broadcasted though(unless its a scalar with 1 element). All that is already checked below.

Reply:
So I should move this lower then and just check for convert?

Reply:
@pfultz2  retesting this , the change doesn't do anything. Removed it.